sympy/series/sequences.py

```diff
@@ -0,0 +1,1123 @@
+        """The ending point of the sequence. This point is included"""
+        raise NotImplementedError("(%s).stop" % self)
+
+    @property
+    def step(self):
+        """Increase points by step"""
+        raise NotImplementedError("(%s).step" % self)
+
+    @property
+    def length(self):
+        """Length of the sequence"""
+        raise NotImplementedError("(%s).length" % self)
+
+    @property
+    def variables(self):
+        """Returns a tuple of variables that are bounded"""
+        return ()
+
+    @property
+    def free_symbols(self):
+        """
+        This method returns the symbols in the object, excluding those
+        that take on a specific value (i.e. the dummy symbols).
+
+        Examples
+        ========
+
+        >>> from sympy import SeqFormula
+        >>> from sympy.abc import n, m
+        >>> SeqFormula((m*n**2, n), (0, 5)).free_symbols
+        set([m])
+        """
+        fsyms = set().union(*[a.free_symbols for a in self.args])
+        for d in self.variables:
+            if d in fsyms:
+                fsyms.remove(d)
+        return fsyms
+
+    @cacheit
+    def coeff(self, pt):
+        """Returns the coefficient at point pt"""
+        if pt < self.start or pt > self.stop:
+            raise IndexError("Index %s out of bounds %s" %(pt, self.interval))
+        return self._eval_coeff(pt)
+
+    def _eval_coeff(self, pt):
+        raise NotImplementedError("The _eval_coeff method should be added to"
+                                  " %s to return coefficient so it is available"
+                                  " when coeff calls it."
+                                  % self.func)
+
+    def _ith_point(self, i, step=None):
+        """
+        Returns the i'th point of a sequence
+        If start point is negative infinity, point is returned from the end.
+        Assumes the first point to be indexed zero.
+
+        Examples
+        =========
+
+        >>> from sympy import oo
+        >>> from sympy.series.sequences import SeqExpr
+
+        bounded
+
+        >>> SeqExpr((1, 2, 3), (-10, 10))._ith_point(0)
+        -10
+        >>> SeqExpr((1, 2, 3), (-10, 10))._ith_point(5)
+        -5
+        >>> SeqExpr((1, 2, 3), (-10, 10, 2))._ith_point(5)
+        0
+
+        End is at infinity
+
+        >>> SeqExpr((1, 2, 3), (0, oo))._ith_point(5)
+        5
+        >>> SeqExpr((1, 2, 3), (0, oo, 2))._ith_point(5)
+        10
+
+        Starts at negative infinity
+
+        >>> SeqExpr((1, 2, 3), (-oo, 0))._ith_point(5)
+        -5
+        >>> SeqExpr((1, 2, 3), (-oo, 0, 2))._ith_point(5)
+        -10
+        """
+        if self.start is S.NegativeInfinity:
+            initial = self.stop
+        else:
+            initial = self.start
+
+        if step is None:
+            if self.start is S.NegativeInfinity:
+                step = -self.step
+            else:
+                step = self.step
+
+        return initial + i*step
+
+    def add(self, other):
```

Comment:
I couldn't really decide on how ``div`` should work. Plus I didn't find much of it's case. I say it will best to implement it incase we get a use for it.